components/List/docs/README.md

```diff
@@ -0,0 +1,244 @@
+# List
+
+Material Design Lists are continuous groups of text and/or images. The [Material guidelines](https://material.io/go/design-lists) for Lists are extensive, and there is no class at this time for implementing any one of them, let alone all of them. However, we are starting to add classes that represent individual List Items. We currently offer two List Item Cells:
+
```

Comment:
Consider adding the following doc tags:

```
<!-- badges -->

<!-- design-and-api -->

<!-- toc -->
```

Reply:
Example: https://raw.githubusercontent.com/material-components/material-components-ios/develop/components/AppBar/docs/README.md